pages/_app.js

```diff
@@ -1,11 +1,69 @@
 import "~/styles/globals.css";
 
+import { ConfigProvider } from "antd";
+import ptPT from "antd/lib/locale/pt_PT";
 import { AuthProvider } from "~/components/Auth";
+import Head from "next/head";
 
 function Shuriken({ Component, pageProps }) {
+  const typeTemplate = "Não é um ${type} válido";
+  const validateMessages = {
+    // You can use '${name}' in these string to replace with the field name
+    default: "Erro de validação",
```

Comment:
This is really nice, in the future we could deal better with i18n using [this](https://github.com/isaachinman/next-i18next).